resources/l10n/de/LC_MESSAGES/loot.po

```diff
@@ -651,32 +655,36 @@ msgstr "Finde Vorheriges"
 msgid "Find Next"
 msgstr "Finde Nächstes"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:109
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:118
 msgid "Groups Editor"
 msgstr "Gruppen-Editor"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:111
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:120
 msgid "Group name"
 msgstr "Gruppen-name"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:112
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:121
 msgid "Add a new group"
 msgstr "Füge neue Gruppe hinzu"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:130
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:139
 msgid "No plugins are in this group."
 msgstr "Keine Plugins sind in dieser Gruppe."
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:142
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:151
 #, c-format
 msgid "Plugins in %s"
 msgstr "Plugins im %s"
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:162
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:172
 msgid "Group already exists!"
 msgstr "Gruppe existiert bereits!"
 
-#: src/gui/qt/plugin_editor/group_tab.cpp:92
+#: src/gui/qt/plugin_editor/group_tab.cpp:56
+msgid "%1 (Group does not exist)"
+msgstr "%1% (Gruppe existiert nicht)"
```

Comment:
The second `%` isn't needed, `%1` is sufficient.

Reply:
Done